.github/workflows/ci.yml

```diff
@@ -78,6 +78,10 @@ jobs:
     if: >
       !contains(github.event.head_commit.message, '[skip tests]') &&
       !contains(github.event.head_commit.message, '[skip build]')
+    strategy:
+      matrix:
+        subproject: [ "bulk-ingester", "common", "common-authentication", "common-test",
+                      "live-ingester", "prediction-applier", "prediction-applier-extension" ]
```

Comment:
`distribution` module is missing

Reply:
there is no tests there, but maybe I should add it 🤔

Reply:
Module build phase is also kind of a test if there are not any compilation/packaging errors